ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/execution/LedgerTimeAwareCommandExecutor.scala

```diff
@@ -95,16 +95,19 @@ final class LedgerTimeAwareCommandExecutor(
       }
   }
 
+  // Does nothing if `t` is empty. This can happen if the transaction only regarded divulged contracts.
   private[this] def advanceOutputTime(
       res: CommandExecutionResult,
-      t: Instant
+      newTime: Option[Instant],
   ): CommandExecutionResult =
-    res.copy(
-      transactionMeta =
-        res.transactionMeta.copy(ledgerEffectiveTime = Time.Timestamp.assertFromInstant(t)))
+    newTime.fold(res)(
+      t =>
+        res.copy(transactionMeta =
+          res.transactionMeta.copy(ledgerEffectiveTime = Time.Timestamp.assertFromInstant(t))))
 
-  private[this] def advanceInputTime(cmd: Commands, t: Instant): Commands =
-    cmd.copy(ledgerEffectiveTime = t)
+  // Does nothing if `t` is empty. This happens if the transaction only regarded divulged contracts.
```

Comment:
```suggestion
  // Does nothing if `newTime` is empty. This can happen if the transaction only regarded divulged contracts.
```

Reply:
Why `can`?

Reply:
Oh, it's in the other comment. That's the wrong one. :wink:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5541/commits/759e5cacf0196ae492974383b47f1400667f1ac5

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5541/commits/abfc4cf353d52e71cce74396ea958554f7f06b30